jerry-core/rcs/rcs-chunked-list.cpp

```diff
@@ -15,208 +15,304 @@
 
 #include "rcs-chunked-list.h"
 
+
+
+/**
+ * Set previous node for the specified node
+ */
+static void
+rcs_chunked_list_set_prev (rcs_chunked_list_node_t *node_p, /**< node to set previous for */
+                              rcs_chunked_list_node_t *prev_node_p) /**< the previous node */
+{
+  JERRY_ASSERT (node_p != NULL);
+
+  MEM_CP_SET_POINTER (node_p->prev_cp, prev_node_p);
+}
+
 /**
- * Constructor
+ * Set next node for the specified node
+ */
+static void
+rcs_chunked_list_set_next (rcs_chunked_list_node_t *node_p, /**< node to set next for */
+                              rcs_chunked_list_node_t *next_node_p) /**< the next node */
+{
+  JERRY_ASSERT (node_p != NULL);
+
+  MEM_CP_SET_POINTER (node_p->next_cp, next_node_p);
+}
+
+/**
+ * Get size of the node
+ *
+ * @return size of node, including header and data space
+ */
+static size_t
+rcs_chunked_list_get_node_size (void)
+{
+  size_t size =
+  mem_heap_recommend_allocation_size (sizeof (rcs_chunked_list_node_t) + 1u);
+
+  JERRY_ASSERT (size != 0 && size >= sizeof (rcs_chunked_list_node_t));
+
+  return size;
+}
+
+/**
+ * Assert that the list state is correct
+ */
+static void
+assert_chunked_list_is_correct (rcs_chunked_list_t *cl_p)
```

Comment:
What is `cl_p`? We usually add describing comment for each function's argument in the form `/**< ... */` or multiline, if necessary:
```
/**< ...
 *   ... */